test/foundry/perpMath/PerpMath.t.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+import "forge-std/Test.sol";
+import "../../../contracts/lib/PerpMath.sol";
+
+contract PerpMathTest is Test {
+    function test_findMedianOfThree() public {
+        assertEq(PerpMath.findMedianOfThree(1, 2, 3), uint256(2));
+        assertEq(PerpMath.findMedianOfThree(2, 1, 3), uint256(2));
+        assertEq(PerpMath.findMedianOfThree(1, 3, 2), uint256(2));
```

Comment:
I think below cases should be tested as well:
2 3 1
3 1 2
3 2 1
above are permutations

below make sure that in the duplication situaiton, the result is what we want
1 1 1
1 2 2

Reply:
fixed in f2db4928c431c5a838bfe1a3f224c510eb418749